plugin-api/src/main/java/org/hyperledger/besu/plugin/data/TransactionType.java

```diff
@@ -18,8 +18,8 @@
 
 public enum TransactionType {
   FRONTIER(0xf8 /* doesn't end up being used as we don't serialize legacy txs with their type */),
-  ACCESS_LIST(0x1),
-  EIP1559(0xf /* placeholder value until we know what the real type byte will be */);
```

Comment:
If we know this is going to be the transaction type, we should remove this comment

Reply:
makes sense